lib/ical/parse.js

```diff
@@ -165,7 +165,16 @@ parse._handleContentLine = function(line, state) {
       throw new ParserError("Invalid parameters in '" + line + "'");
     }
     params = parsedParams[0];
-    lastParamIndex = parsedParams[1].length + parsedParams[2] + paramPos;
+    // Handle parameter values with multiple entries
+    let parsedParamLength;
+    if (typeof parsedParams[1] === 'string') {
+      parsedParamLength = parsedParams[1].length;
+    } else {
+      parsedParamLength = parsedParams[1].reduce((accumulator, currentValue) => {
```

Comment:
I'm suspecting we do most of the calculation for parameters within `_parseParameters` already and would like to see if we can optimize performance here a bit.

Could you instead use those numbers and return them as needed? It looks like `parsedParameters[2]` is only used to calculate `lastParamIndex`, so we could change the meaning of this field in the array to include the parsed param length.

Reply:
While you are at it, can you update the return value in jsdoc for `_parseParameters` ? It simply says an object with key/value right now.